libbeat/publisher/queue/diskqueue/segments.go

```diff
@@ -170,9 +170,17 @@ func (s bySegmentID) Less(i, j int) bool { return s[i].id < s[j].id }
 // Scan the given path for segment files, and return them in a list
 // ordered by segment id.
 func scanExistingSegments(logger *logp.Logger, pathStr string) ([]*queueSegment, error) {
-	files, err := ioutil.ReadDir(pathStr)
+	entries, err := os.ReadDir(pathStr)
```

Comment:
What about `segmentFiles`? I don't like `entries` here because it's overloading the primary meaning of "entry" here which is `queue.Entry` representing an individual object in the queue, whereas these files each represent a segment containing many entries.

Reply:
I'll update it.